io/io/src/TBufferMerger.cxx

```diff
@@ -69,7 +68,7 @@ void TBufferMerger::WriteOutputFile()
       merger.OutputFile(fName.c_str(), fOption.c_str(), fCompress);
    }
 
-   while (!done) {
+   for (;;) {
```

Comment:
> but compilers may issue a warning for while(true) ("constant expression used as loop expression")

Is that speculation or observation?  It is such a common pattern that it seems like a bug in the compiler to complain about that one ...